08-projects.Rmd

```diff
@@ -6133,6 +6133,31 @@ The following questions will use the dataset found in Scholar:
 
 A public sample of the data can be found [here](https://www.datadepot.rcac.purdue.edu/datamine/data/movies-and-tv/imdb.db).
```

Comment:
Hi @kevinamstutz , could you double check and correct the link here?

Reply:
I'm removing the link. I need to take the time to get a good sample of the database which is tough to do and keep in tact some of the relationships. For now, I think its just best to remove it. Removing it from P15 too.

Reply:
Indeed removing the link would makes more sense - as opening a database should not take too much time comparing to loading a large dataset in R, and students can already explore the contents with the actual database. Thanks @kevinamstutz for the update!